client/components/card/index.jsx

```diff
@@ -40,7 +40,7 @@ export default React.createClass( {
 
 		return React.createElement(
 			element,
-			assign( {}, this.props, { className } ),
+			assign( {}, omit( this.props, 'compact' ), { className } ),
```

Comment:
`omit` returns a new object, so you can skip an unnecessary object clone by leaving out the first empty object argument.